### PR TITLE
Dependency Updates

### PR DESCRIPTION
**Pull request by nateridderman**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -109,33 +108,30 @@ dependencies {
...
     androidTestImplementation "androidx.compose.ui:ui-test-junit4:$composeVersion"
     kapt "com.google.dagger:hilt-android-compiler:$hilt_version"
-    implementation 'androidx.hilt:hilt-lifecycle-viewmodel:1.0.0-alpha03'
     kapt 'androidx.hilt:hilt-compiler:1.0.0'
 
-
     // Database
-    implementation "androidx.room:room-runtime:2.2.5"
-    kapt "androidx.room:room-compiler:2.2.5"
-    implementation 'com.facebook.stetho:stetho:1.5.1'
```
Comment (timmiller17) [inline app/build.gradle:128]: Did you decide this wasn't needed or was it broken? Just curious. :)

**Review by timmiller17 — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (nateridderman) [reply app/build.gradle:128]: I think there are better alternatives to Stetho these days, most things you can see with just Android Studio features.

Comment (nateridderman) [reply app/build.gradle:128]: It wasn't broken... I added a few "cleanup" things to the PR

**Comment by nateridderman** [comment]
@rorpage are you able to test it? I hope it builds well for you and the app runs well.

**Comment by wsarles** [comment]
Looks good to me.

**Comment by rorpage** [comment]
This is so awesome! Thanks @nateridderman!! Merging now

**Pull request merged**